src/app/static/src/app/components/downloadResults/DownloadResults.vue

```diff
@@ -167,11 +162,6 @@
             getUserCanUpload: mapActionByName("adr", "getUserCanUpload"),
             getUploadFiles: mapActionByName("adrUpload", "getUploadFiles"),
             prepareOutputs: mapActionByName("downloadResults", "prepareOutputs"),
-            downloadComparisonReport: mapActionByName("downloadResults", "downloadComparisonReport"),
-            downloadSpectrumOutput: mapActionByName("downloadResults", "downloadSpectrumOutput"),
-            downloadSummaryReport: mapActionByName("downloadResults", "downloadSummaryReport"),
-            downloadCoarseOutput: mapActionByName("downloadResults", "downloadCoarseOutput"),
-            downloadAgywTool: mapActionByName("downloadResults", "downloadAgywTool"),
```

Comment:
Normally you could remove all these actions and their tests, but let's also do this after Mantra's changes as otherwise I think it will clash